ui/components/reservation/ReservationInfoCard.tsx

```diff
@@ -112,12 +112,18 @@ const ReservationInfoCard = ({
 
   const price: string =
     reservation?.state === "REQUIRES_HANDLING"
-      ? getReservationUnitPrice(reservationUnit, begin)
+      ? getReservationUnitPrice(reservationUnit, new Date(begin))
       : getReservationPrice(reservation?.price, t("prices:priceFree"));
 
   const shouldDisaplyTaxPercentage: boolean =
     reservation?.state === "REQUIRES_HANDLING"
-      ? getReservationUnitPrice(reservationUnit, begin, 0, false, true) !== "0"
+      ? getReservationUnitPrice(
+          reservationUnit,
+          new Date(begin),
```

Comment:
Need to check truthiness.

Reply:
Is it needed here? according to 

`const { begin, end, taxPercentageValue } = reservation || {};`

begin is always string? or do you mean is it a invalid date?